app/hurl.ml

```diff
@@ -60,6 +63,12 @@ let header =
   let doc = "HTTP header (key:value)" in
   Arg.(value & opt_all header_c [] & info [ "header" ] ~doc ~docv:"HEADER")
 
+let protocol =
+  let vs =
+    [ Some `HTTP_1_1, Arg.info [ "only-http-1-1" ] ~doc:"Use the HTTP/1.1 protocol"
+    ; Some `H2, Arg.info [ "only-h2" ] ~doc:"Use the H2 protocol" ] in
```

Comment:
```suggestion
    [ Some `HTTP_1_1, Arg.info [ "http1" ] ~doc:"Only use the HTTP/1.1 protocol"
    ; Some `H2, Arg.info [ "http2" ] ~doc:"Only use the H2 protocol" ] in
```

let's use `--http1` and `--http2` and make the docstring slightly more clear. :)